atlp-rwanda/destructors-ec-bn
# #184693449 verify user via email

#### What does this PR do?

- [x] A User  should be verified via email:
  - After signup the user receive an verification email
  - After verification the user allowed to login 
  - if user login with google  there is no need to verify


#### Description of Task to be completed?

- [x] A User  should be verified via email:

   -A user should be verified  via email
 
#### How should this be manually tested?

- Clone the repository 
- Change the directory to destructors-ec-bn by running `cd destructors-ec-bn` in you terminal
- Checkout ft-user-email-verify-184693449 branch by running `git checkout ft-user-email-verify-184693449` in you terminal
- open .env.example to check how you can create you .env file 
- Create your .env file
- Then install dependencies and devdependencies used by running `npm install` in you terminal
- To create the models used in this project run `npm migrate`
- Run the server by running `npm run dev` in your terminal
- open you browser and in URL field type http://localhost:PORT/api-docs (the PORT is the port you have written in your .env file. The default port is 3000)
- swagger page will open the navigate under  the products tag 
- you can post all user using /api/v1/signup and go to the gmail user and then click to the link so as to verify

#### Any background context you want to provide?

#### What are the relevant pivotal tracker/Trello stories?

#184637449


#### Screenshots (if appropriate)

#### how token should be written

![Screenshot (72)](https://user-images.githubusercontent.com/101397608/228195707-899e9915-bb17-4526-ab47-dbc36e21379a.png)


#### Questions:
